tests/tiappxml_test.js

```diff
@@ -2,7 +2,8 @@ const should = require('should'); // eslint-disable-line no-unused-vars
 
 const ti = require('../lib/titanium'),
 	fs = require('fs'),
-	path = require('path');
+	path = require('path'),
+	os = require('os');
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/tiappxml_te...
  
  
  DangerID: danger-id-Danger;
  File: tests/tiappxml_test.js;
  Line: 6;
-->

- :warning: tests/tiappxml_test.js line 6 – 'os' is assigned a value but never used. (no-unused-vars)